lib/oauth2-client.js

```diff
@@ -68,15 +67,10 @@ async function _getUncachedAccessToken({
   client_id, client_secret, token_endpoint, grant_type, scope, audience,
   maxRetries
 }) {
-  let {token} = await storage.get({client_id, token_endpoint}) || {};
-  if(!(token && token.access_token)) {
-    token = await getNewAccessToken({
-      client_id, client_secret, token_endpoint, grant_type, scope, audience,
-      maxRetries
-    });
-    await storage.insert({token});
-  }
-  return token.access_token;
+  return getNewAccessToken({
```

Comment:
It appears to me that this helper `_getUncachedAccessToken` is no unneeded.  It is now just providing an unnecessary wrapper around `getNewAccessToken`, is that correct?

Reply:
Yes, updated.